### DISCUSSION TITLE
Star Ratings

### DISCUSSION THREAD

**Discussion by TyCathcart** [ByteLegend/ByteLegend#140] [Q&A]
How does the game decide how many stars to award for a challenge? I'm stuck with one star on "Method and Field" and would like to know if I am doing something wrong. Thanks!

**Comment by gtn1024** [comment]
Did you have seen the comments by the bot? 

<https://github.com/ByteLegendQuest/java-write-static-methods/pull/191>

![image](https://user-images.githubusercontent.com/69113276/184261969-986b3978-b9b7-4f52-a952-efd7e2ffecd0.png)
